src/V3Undriven.cpp

```diff
@@ -523,10 +548,39 @@ class UndrivenVisitor final : public VNVisitorConst {
         iterateChildrenConst(nodep);
         if (nodep->keyword() == VAlwaysKwd::ALWAYS_COMB) UINFO(9, "   Done " << nodep);
     }
+
     void visit(AstNodeFTaskRef* nodep) override {
         VL_RESTORER(m_inFTaskRef);
         m_inFTaskRef = true;
+
         iterateChildrenConst(nodep);
+
+        if (!m_capturep) return;
+
+        // If writeSummary is enabled, task/function definitions are treated as non-executed.
+        // Do not apply writeSummary at calls inside a task definition, or they will look like
+        // independent drivers (phantom MULTIDRIVEN). Did the lambda on purpose - lessen chance of
+        // screwup in future edits.
+
+        const auto inExecutedContext = [this]() {
+            return !(m_taskp && !m_alwaysp && !m_inContAssign && !m_inInitialStatic && !m_inBBox
+                     && !m_taskp->dpiExport());
+        };
+
+        if (!inExecutedContext()) return;
```

Comment:
No need for Lambda unless I'm missing something.

```suggestion
        const bool inExecutedContext = 
             !(m_taskp && !m_alwaysp && !m_inContAssign && !m_inInitialStatic && !m_inBBox
                     && !m_taskp->dpiExport());
        if (!inExecutedContext) return;
```

Reply:
Fixed.  Was a holdover as I was thinking I was going to need this in multiple places.